src/x509.c

```diff
@@ -7517,6 +7517,464 @@ WOLFSSL_X509_CRL* wolfSSL_d2i_X509_CRL(WOLFSSL_X509_CRL** crl,
 
     return newcrl;
 }
+
+WOLFSSL_X509_NAME* wolfSSL_X509_CRL_get_issuer_name(WOLFSSL_X509_CRL* crl)
+{
+    WOLFSSL_ENTER("X509_CRL_get_issuer_name");
+    if (crl){
+        return crl->crlList->issuer;
+    }
+    return NULL;
+}
+
+int wolfSSL_X509_CRL_version(WOLFSSL_X509_CRL* crl)
+{
+    if (crl == NULL)
+        return 0;
+    return crl->crlList->version;
+}
+
+int wolfSSL_X509_CRL_get_signature_type(WOLFSSL_X509_CRL* crl)
+{
+    if (crl == NULL)
+        return 0;
+    return crl->crlList->signatureOID;
```

Comment:
What if `crl->crlList == NULL`?

Reply:
lets add a check for crl being null too here before the dereferencing it to check crlList since this is a public facing function